### PR TITLE
BSTOR-40 - extract brand picker

### PR DESCRIPTION
**Pull request by Pacane**

### CODE REVIEW

File: client/src/main/java/com/arcbees/beeshop/client/application/widget/brandpicker/BrandPicker.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+import com.google.gwt.uibinder.client.UiBinder;
+import com.google.gwt.uibinder.client.UiField;
+import com.google.gwt.uibinder.client.UiTemplate;
+import com.google.gwt.user.client.ui.IsWidget;
+import com.google.gwt.user.client.ui.Widget;
+import com.gwtplatform.mvp.client.proxy.PlaceManager;
+import com.gwtplatform.mvp.shared.proxy.PlaceRequest;
+
+public class BrandPicker implements IsWidget, AttachEvent.Handler {
+    @UiTemplate("BrandPicker.ui.xml")
```
Comment (jclariviere) [inline client/src/main/java/com/arcbees/beeshop/client/application/widget/brandpicker/BrandPicker.java]: ?

File: client/src/main/java/com/arcbees/beeshop/client/application/widget/brandpicker/HomeBrandPicker.java [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import com.google.gwt.uibinder.client.UiTemplate;
+import com.google.gwt.user.client.ui.IsWidget;
+import com.gwtplatform.mvp.client.proxy.PlaceManager;
+
+public class HomeBrandPicker extends BrandPicker {
+    @UiTemplate("HomeBrandPicker.ui.xml")
+    interface Binder extends UiBinder<IsWidget, HomeBrandPicker>{
+    }
+
+
```
Comment (jclariviere) [inline client/src/main/java/com/arcbees/beeshop/client/application/widget/brandpicker/HomeBrandPicker.java]: rm nl

**Comment by jclariviere** [comment]
LGTM

File: client/src/main/java/com/arcbees/beeshop/client/application/widget/brandpicker/BrandPicker.java (hunk 1 above)
Comment (Pacane) [inline client/src/main/java/com/arcbees/beeshop/client/application/widget/brandpicker/BrandPicker.java]: Yeah don't need that anymore.

**Pull request merged**